sympy/printing/latex.py

```diff
@@ -786,8 +786,16 @@ def _print_Integral(self, expr):
 
                 symbols.insert(0, r"\, d%s" % self._print(symbol))
 
+        if any(_coeff_isneg(i) for i in expr.args):
+            return r"%s %s%s" % (tex, self.parenthesize(expr.function,
+                                                    PRECEDENCE["Mul"],
+                                                    is_neg=True,
+                                                    strict=True),
+                             "".join(symbols))
+
         return r"%s %s%s" % (tex, self.parenthesize(expr.function,
                                                     PRECEDENCE["Mul"],
+                                                    is_neg=False,
```

Comment:
This could just be `is_neg=any(_coeff_isneg(i) for i in expr.args)`.

Reply:
Yes, i committed again with the changes.